.github/workflows/release.yml

```diff
@@ -1,17 +1,23 @@
 name: "release"
 on:
   push:
-    tags:
-      - '*'
+    tags: # Will upload to luarocks.org
+      - "*"
+  pull_request: # Will test a local install without uploading to luarocks.org
+  
 jobs:
   luarocks-upload:
     runs-on: ubuntu-latest
     steps:
-      - uses: nvim-neorocks/luarocks-tag-release@v2.2.0
+      - uses: actions/checkout@v3
+      - uses: nvim-neorocks/luarocks-tag-release@v5
         env:
           LUAROCKS_API_KEY: ${{ secrets.LUAROCKS_API_KEY }}
         with:
+          name: nvim-treesitter
+          version: 0.1
+          license: apache 2
           detailed_description: |
             The goal of nvim-treesitter is both to provide a simple and easy way to use the interface for tree-sitter in Neovim
             and to provide some basic functionality such as highlighting based on it.
-          build_type: "make"
+          template: nvim-treesitter-luarocks.template
```

Comment:
this file should not live on the top level

Reply:
sure. Should I create a "contrib" folder and move it there or to "scripts" ?

Reply:
I'd prefer if the luarocks files live in `contrib`, yes -- makes their status clear.